troposphere/__init__.py

```diff
@@ -648,8 +648,8 @@ def to_json(self, indent=4, sort_keys=True, separators=(',', ': ')):
         return json.dumps(self.to_dict(), indent=indent,
                           sort_keys=sort_keys, separators=separators)
 
-    def to_yaml(self, clean_up=False, long_form=False):
-        return cfn_flip.to_yaml(self.to_json(), clean_up=clean_up,
+    def to_yaml(self, clean_up=False, long_form=False, sort_keys=True):
+        return cfn_flip.to_yaml(self.to_json(sort_keys), clean_up=clean_up,
```

Comment:
I'm not sure this would work. Shouldn't that be a named param like this (otherwise it would match the indent param):
```
return cfn_flip.to_yaml(self.to_json(sort_keys=sort_keys), clean_up=clean_up,
```